packages/editor/src/components/document-outline/item.js

```diff
@@ -17,8 +12,9 @@ const TableOfContentsItem = ( {
 	children,
 	isValid,
 	level,
-	onClick,
 	path = [],
+	target,
```

Comment:
Why is this named target instead of just `href`, seems a bit confusing as target has a different meaning for links?

Reply:
ah, good point - i'll change this to href

Reply:
changed in https://github.com/WordPress/gutenberg/pull/10815/commits/b85b4c9c942fe618b60b13baebdf2321d602b801